File: src/main/java/org/neonlabs/e2e/go-eth/events_test.go
package go_eth

import (
	"testing"

	log "github.com/sirupsen/logrus"

	"github.com/dailymotion/allure-go"
	"github.com/stretchr/testify/assert"
)

func TestNewBlocksSubscription(t *testing.T) {
	t.Parallel()
	allure.SkipTest(t,
		allure.Epic(FeatureEvents),
		allure.Feature("Subscription to new blocks"),
		// allure.Story("Subscription to new blocks"),
		allure.Description("Subscription to new blocks"),
		allure.Action(func() {
			log.Println("111")
			assert.Equal(t, 1, 1, "111")
		}))
}
func TestEventLogSubscription(t *testing.T) {
	t.Parallel()
	allure.SkipTest(t,
		allure.Epic(FeatureEvents),
		allure.Feature("Subscription to event logs"),
		// allure.Story("Subscription to event logs"),
		allure.Description("Subscription to event logs"),
		allure.Action(func() {
			log.Println("111")
			assert.Equal(t, 1, 1, "111")
		}))
}
func TestReadEventLogs(t *testing.T) {
	t.Parallel()
	allure.SkipTest(t,
		allure.Epic(FeatureEvents),
		allure.Feature("Read event logs"),
		// allure.Story("Read event logs"),
		allure.Description("Read event logs"),
		allure.Action(func() {
			log.Println("111")
			assert.Equal(t, 1, 1, "111")
		}))
}
func TestReadERC20TokenEventLogs(t *testing.T) {
	t.Parallel()
	allure.SkipTest(t,
		allure.Epic(FeatureEvents),
		allure.Feature("Read ERC20 token event logs"),
		// allure.Story("Read ERC20 token event logs"),
		allure.Description("Read ERC20 token event logs"),
		allure.Action(func() {
			log.Println("111")
			assert.Equal(t, 1, 1, "111")
		}))
}
func TestRead0xProtocolEventLogs(t *testing.T) {
	t.Parallel()
	allure.SkipTest(t,
		allure.Epic(FeatureEvents),
		allure.Feature("Read 0x protocol event logs"),
		// allure.Story("Read 0x protocol event logs"),
		allure.Description("Read 0x protocol event logs"),
		allure.Action(func() {
			log.Println("111")
			assert.Equal(t, 1, 1, "111")
		}))
}

File: src/test/java/org/neonlabs/e2e/web3j/EventsTest.java
package org.neonlabs.e2e.web3j;

import static org.junit.jupiter.api.Assertions.assertAll;
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;
// import static org.neonlabs.e2e.web3j.Constants.Epic;
import static org.neonlabs.e2e.web3j.Constants.FeatureEvents;
import static org.neonlabs.e2e.web3j.Constants.NotYetDone;

import io.qameta.allure.Description;
import io.qameta.allure.Epic;
// import io.qameta.allure.Feature;
import io.qameta.allure.Story;
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

/**
 * Events tests.
 */
@Epic(FeatureEvents)
// @Feature(FeatureEvents)
@DisplayName(FeatureEvents)
@SuppressWarnings("PMD.JUnitTestContainsTooManyAsserts")
class EventsTest {
  @Test
  @Story("Subscription to new blocks")
  @Description("Subscription to new blocks")
  @Disabled(NotYetDone)
  void test01() {
    assertAll(
        () -> assertNotNull(1, ""),
        () -> assertEquals(1, 1, ""));
  }

  @Test
  @Story("Subscription to event logs")
  // @DisplayName(FeatureEvents)
  @Description("Subscription to event logs")
  @Disabled(NotYetDone)
  void test02() {
    assertAll(
        () -> assertNotNull(1, ""),
        () -> assertEquals(1, 1, ""));
  }

  @Test
  @Story("Read event logs")
  // @DisplayName(FeatureEvents)
  @Description("Read event logs")
  @Disabled(NotYetDone)
  void test03() {
    assertAll(
        () -> assertNotNull(1, ""),
        () -> assertEquals(1, 1, ""));
  }

  @Test
  @Story("Read ERC20 token event logs")
  // @DisplayName(FeatureEvents)
  @Description("Read ERC20 token event logs")
  @Disabled(NotYetDone)
  void test04() {
    assertAll(
        () -> assertNotNull(1, ""),
        () -> assertEquals(1, 1, ""));
  }

  @Test
  @Story("Read 0x protocol event logs")
  // @DisplayName(FeatureEvents)
  @Description("Read 0x protocol event logs")
  @Disabled(NotYetDone)
  void test05() {
    assertAll(
        () -> assertNotNull(1, ""),
        () -> assertEquals(1, 1, ""));
  }
}
